### PR TITLE
SBS consolidation, mooted invites

### PR DESCRIPTION
**Pull request by zapu**

### CODE REVIEW

File: go/teams/service_helper.go [hunk 1]
```diff
@@ -1114,6 +1114,15 @@ func removeInviteID(ctx context.Context, team *Team, invID keybase1.TeamInviteID
 	return team.postTeamInvites(ctx, invites)
 }
 
+func mootInviteID(ctx context.Context, team *Team, invID keybase1.TeamInviteID, uv keybase1.UserVersion) error {
```
Comment (maxtaco) [inline go/teams/service_helper.go]: move this code over to `handler`?

**Comment by zapu** [comment]
@maxtaco can you take a look? I addressed earlier comment, merged serverside, and the CI passed now.

**Comment by maxtaco** [comment]
Yes, I will. It's in a queue behind several other reviews. Trying my best to get to them all.

File: go/teams/handler.go [hunk 2]
```diff
@@ -185,6 +185,14 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 		}
 
 		if currentRole.IsOrAbove(invite.Role) {
+			if team.IsImplicit() {
```
Comment (maxtaco) [inline go/teams/handler.go]: Can't this same mechanism work for explicit teams?

**Comment by zapu** [comment]
@mlsteele would you mind chiming in on this issue? once we release this, it will be in sigchains forever, so it's important to do the right play here. Thanks!

**Comment by maxtaco** [comment]
Let's please try to land this today, it's really dragged on a long time.

**Pull request merged**